### DISCUSSION TITLE
v0.1.0

### DISCUSSION THREAD

**Discussion by jpsyx** [tsdataclinic/data-gather#282] [Announcements]
This is the first release of Data Gather!

## What's Changed
* Implemented the entire app.

In particular, thanks to:
* @tiffanymeits for the initial design and your contributions to setting up FastAPI, Azure auth, and the UI implementation to create new form stages
* @indraneel for your contributions to the Airtable authentication
* @kbiggers for setting up the backend python models with SQLModel and all your various bug and usability fixes throughout the app, no matter the level of effort
* @brahmcapoor for your contributions to setting up an Airtable service, setting up the backend models to support multiple languages, and your various bug fixes along the way
* @iebaker for building the engine that can process an Interview schema and render a UI for it

And of course, thanks to [Wellfare](https://www.wellfare.org/) for being excellent partners and being the motivation to building out this 

**Full Changelog**: https://github.com/tsdataclinic/data-gather/commits/v0.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/tsdataclinic/data-gather/releases/tag/v0.1.0'>v0.1.0</a>.</em>

**Comment by els2171** [comment]
Congratulations, team!

**Comment by oxcar** [comment]
I was not able to install it following istructions. Is it in active development?

**Reply by jpsyx** [reply]
Hi, thanks for checking! Yes this is still in development. Could you let me know what commands you ran and what errors you got?

**Reply by oxcar** [reply]
1. Created a new virtualenv (3.10). 
2. Installed requirements `pip install -r requirements.txt`
3. Created the database `yarn db-upgrade`
4. Crashes when running `yarn api`

`INFO:fastapi_azure_auth:Fetching OpenID Connect config from https://.b2clogin.com/.onmicrosoft.com//v2.0/.well-known/openid-configuration`
`ERROR:fastapi_azure_auth:Unable to fetch OpenID configuration from Azure AD. Error: [Errno 8] nodename nor servname provided, or not known`

**Reply by jpsyx** [reply]
Oh I see! Sorry about that. Currently the codebase is tightly coupled to requiring an Azure AD B2C instance deployed, so you'd have to set up your own Azure AD B2C instance and add those environment variables. I'll update the readme on how to do that. But in the next week or two we'll be releasing a new version that lets you bypass auth for local development, so you won't need any Azure AD B2C env vars to develop locally anymore. I'll respond back on this thread once that update is done and you should be able to run things locally!

**Reply by oxcar** [reply]
Understood.
